File: isUnique.js
function isUnique(str) {
    if (str.length > 128) return false;
    let strObj = {};
    let arr = str.split("");
    for (let i = 0; i < str.length; i++) {
      if (arr[i] in strObj) {
        return false;
      }
      strObj[arr[i]] = i;
    }
    return true;
  }
  
  console.log(isUnique("awefaweaa"));
  

File: isUnique.py
def isUnique(str):
    if len(str) > 128:
        return False   
    strObj = {}
    for x in range(len(str)):
        if str[x] in strObj:
            return False
        else:
            strObj[str[x]] = x
    return True

print(isUnique("hhu"))
